crates/debug/examples/fib-wasm.c

```diff
@@ -0,0 +1,13 @@
+// Compile with:
+//   clang --target=wasm32 fib-wasm.c -o fib-wasm.wasm -g \
+//     -Wl,--no-entry,--export=fib -nostdlib -fdebug-prefix-map=$PWD=.
```

Comment:
Would it be possible to require that `clang` be installed on CI instead of checking in the wasm file?

Reply:
clang for the wasm32 target is not stable to produce consistent DWARF info yet (maybe after LLVM10) on all platforms